<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendPacket</class>
 <widget class="QMainWindow" name="SendPacket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GWakeonLAN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target MAC Address:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mac_in">
        <property name="text">
         <string>00-00-00-00-00-00</string>
        </property>
        <property name="maxLength">
         <number>17</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="send_btn">
        <property name="text">
         <string>Send Magic Packet</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_load">
        <property name="toolTip">
         <string>Load MAC addresses from a backing store</string>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="port_in">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>9</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionBrowse_MySQL_Database"/>
    <addaction name="actionListen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWhatsThis"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionBrowse_MySQL_Database">
   <property name="text">
    <string>Get MACs from a Database</string>
   </property>
   <property name="toolTip">
    <string>Connect to MySQL Database to find MAC addresses</string>
   </property>
  </action>
  <action name="actionListen">
   <property name="text">
    <string>Listen for Magic Packets</string>
   </property>
  </action>
  <action name="actionWhatsThis">
   <property name="text">
    <string>What's This?</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>send_btn</sender>
   <signal>clicked()</signal>
   <receiver>SendPacket</receiver>
   <slot>send_packet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_load</sender>
   <signal>clicked()</signal>
   <receiver>SendPacket</receiver>
   <slot>open_db_browser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>send_packet()</slot>
  <slot>open_db_browser()</slot>
 </slots>
</ui>
